docs/pre-release/releaseNotes.yml

```diff
@@ -32,7 +32,7 @@ changelog: https://github.com/telepresenceio/telepresence/blob/$branch$/CHANGELO
 items:
   - version: 2.y.z
     date: 'TBD'
-    notes:
+    notes: []
```

Comment:
Should the rest of this commit (except for this change) be targetted towards master, which we can then rebase `cycle5-2021` on top of, and then add this to the `cycle5-2021` branch? 

I only say that bc this branch likely won't be merged until the end of the cycle (or at least until we have all the features done).

Reply:
👍  fair enough!